DFC.ServiceTaxonomy.CompUi/Handlers/CacheHandler.cs

```diff
@@ -147,27 +147,38 @@ public async Task ProcessEventGridMessage(Processing processing, ContentEventTyp
 
         if (current == null)
         {
-            _logger.LogError($"Current content is null for following content content type:{processing.ContentType}");
+            _logger.LogError($"Current content is null for following content type:{processing.ContentType}");
```

Comment:
Nit-pick: Maybe add a space after the colon, and maybe surround the content type with '. Up to you.